onchain/rollups/test/foundry/dapp/CartesiDApp.t.sol

```diff
@@ -184,14 +184,14 @@ contract CartesiDAppTest is TestBase {
             LibOutputValidation.getBitMaskPosition(
                 proof.validity.outputIndexWithinInput,
                 _inputIndex
-            )
+            ),
+            abi.encodePacked(uint(1))
```

Comment:
Considering `transfer` returns a boolean, it makes more sense to encode `true` rather than `1`, although they have the same ABI-encoding. Also, return data is encoded with `abi.encode` (although in this case, since there are no variable-length values, it won't make a difference).

```suggestion
            abi.encode(true)
```